Charades_Game_In_JavaScript/Charades Game In JavaScript final blog.md

```diff
@@ -0,0 +1,348 @@
+
+Then, we would use `setInterval()` function to let the timer count down at each second (=1000 milliseconds), and update the variable `seconds` by using the decrementing operator `seconds--;`, and also, update the `textContent` with the decremented seconds or whatever words you want (like "Ready??" "Go!!").
+
+We use an `if-else if-else` loop to separate conditions on the seconds. The seconds would go into the `else` statement after 5 iterations, and we would `clearInterval(count);` and jump to the corresponding page.
+
+![](https://i.imgur.com/k6gd2dv.png)
+
+Now try out the new feature you got for your limiting time version! 
+
+### **Phase 3: Copy to Both Functions**
+
+##### **in JavaScript:**
+Simply copy and paste all the codes we added in the function `time()` in phase 1 and 2 into the function `word()`. Remember to change the redirecting page into `"card_word.html"`!
+
+Now, try clicking both of the buttons!
+
+
+
+
+## Creating Animated Word Cards
+![](https://i.imgur.com/9XlNjfr.png)
+
+Now it is the exciting part to create flipping cards with words on them!
+Let's start with creating the elements in HTML.
+
+##### **in HTML:**
+The HTML of both versions are exactly the same, besides the different JavaScript they included.
+
+There will be a simple `<div>` holding the card which has a `front-face`, and a `back-face`. As learned from Marina Ferreira's blog, I adapted her procedure to create the flipping card. And in the end, I add a timer to give a reference to the players.
+
+```
+<div class="cards" id="words">
+    <p class="front-face" id="word2">dog</p>
+    <p class="back-face" id="word1">pig</p>
+</div>
+<span id="timer"></span>
+```
+##### **in CSS:**
+Since both versions have the same HTML elements, we do not need to create separate CSS files for each design (but you can if you want the cards look different in different versions).
+Besides the design of cards, there are some important features to add to make the card flips as we want. I mainly adapted these [CSS codes](https://github.com/code-sketch/memory-game/blob/master/video-11/styles.css) from [Marina](https://medium.com/free-code-camp/vanilla-javascript-tutorial-build-a-memory-game-in-30-minutes-e542c4447eae), because she has made the flipping effects very fluent and nice. However, instead of flipping it vertically, I changed to horizontally, so we have `transform: rotateX(180deg);`
+
+You can find my full CSS code [here](https://github.com/bitprj/Charades_Game_In_JavaScript/blob/master/finish_code/cardstyle.css).
+
+![](https://i.imgur.com/r8J1YZg.gif)
+
+
+Note: Here is a little tricky. Since I used the same CSS codes from Marina, the `back-face` is shown in the beginning as the first word, so I named it with `id="word1"`. I'll refer to `word1`, `word2` instead to make it more clear.
+
+
+### "Limiting Time" Ver.
+In this version, we want to have a timer to keep track of the time that each card has been shown, and flip the card when the time is up.
```

Comment:
I see now that you mean the version of how you play the game. Please make this clear from the start when you start talking about versions